opensearch-project/OpenSearch-Dashboards
# [Workspace]Replace connections copy with data sources

### Description

This PR is for replacing "connection" copy with "data sources" in workspace creation page like screenshots below.

## Screenshot


![image](https://github.com/user-attachments/assets/f73e3732-6e91-49c3-8854-54bc6f393610)
![image](https://github.com/user-attachments/assets/863ba19d-e151-4247-b597-eb5fb8c95842)
![image](https://github.com/user-attachments/assets/7e59c691-0d56-46d4-8d41-64fbe6881252)


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version loose`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Login with admin user and go to workspace create page by bottom left menu
- Scroll to the "Add data sources" section should show "data sources" instead "connections"
- Click "Add OpenSearch data sources" should show modal like screenshots above
- Click "Add Direct query data sources" should show modal like screenshots above


## Changelog

- feat: [Workspace]Replace connections copy with data sources

### Check List
